classes/Tools.php

```diff
@@ -1080,8 +1080,16 @@ public static function htmlentitiesDecodeUTF8($string)
         return html_entity_decode((string) $string, ENT_QUOTES, 'utf-8');
     }
 
+    /**
+     * @deprecated Since 1.7.9.0.
+     */
     public static function safePostVars()
     {
+        @trigger_error(
+            'Tools::safePostVars() is deprecated since version 1.7.9.0. ',
```

Comment:
```suggestion
            'Tools::safePostVars() is deprecated since version 1.7.9.0.',
```
Little un-needed extra white space 😉 

It's a nitpick so I remove it for you 👍